bin/ncp/BACKUPS/nc-rsync-auto.sh

```diff
@@ -28,10 +28,12 @@ configure()
     return 1;
   }
 
+  # Check if the ssh access is properly configured. For this purpose the command : or echo is called remotely.
+  # If one of the commands works, the test is successful.
   [[ "$DESTINATION" =~ : ]] && {
     local NET="$( sed 's|:.*||' <<<"$DESTINATION" )"
     local SSH=( ssh -o "BatchMode=yes" -p "$PORTNUMBER" "$NET" )
-    ${SSH[@]} : || { echo "SSH non-interactive not properly configured"; return 1; }
+    ${SSH[@]} : || ${SSH[@]} echo || { echo "SSH non-interactive not properly configured"; return 1; }
```

Comment:
```suggestion
    ${SSH[@]} true || { echo "SSH non-interactive not properly configured"; return 1; }
```

would this work? 

thanks for the code :)

Reply:
No, unfortunately this doesn't t work either. I think the reason is, that : and true are both bash builtins, but rsync.net supports only [a specific set of commands](https://www.rsync.net/resources/howto/remote_commands.html).

Reply:
I see, ok then let's just leave the `echo`

Reply:
I added it just leaving the first echo. Many thanks!